emails/templates/private-cloud/RequestRejection.tsx

```diff
@@ -11,7 +11,6 @@ interface EmailProp {
 }
 export const RequestRejectionTemplate = ({ productName, comment }: EmailProp) => {
   if (!productName) return <></>;
-
```

Comment:
to reduce the unnecessary code changes and also for readability, let's keep this new line.